.github/workflows/releaseChart.yaml

```diff
@@ -21,22 +28,58 @@ jobs:
           REMOTE="$(md5sum < index.yaml.remote)"
           echo "$LOCAL" = "$REMOTE"
           test "$LOCAL" = "$REMOTE"
+
   chart-release:
+    name: Create chart release
     runs-on: ubuntu-latest
     needs: [ validate-gh-pages-index ]
+    outputs:
+      version: ${{ steps.version.outputs.next-version }}
+      image-tag: ${{ steps.image-tag.outputs.tag }}
     steps:
+      #TODO Add a check to be sure we are running over main
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-
+          ref: main
+      - name: Add newrelic repository
+        run: helm repo add newrelic https://helm-charts.newrelic.com
+      - uses: newrelic/release-toolkit/validate-markdown@v1
+      - uses: newrelic/release-toolkit/generate-yaml@v1
+        with:
+          included-dirs: ${{ env.CHART_DIRECTORY }}
+          markdown: ${{ env.CHART_DIRECTORY }}/CHANGELOG.md
+          tag-prefix: ${{  env.CHART_TAG_PREFIX }}
+      - uses: newrelic/release-toolkit/is-held@v1
+      - uses: newrelic/release-toolkit/link-dependencies@v1
+        with:
+          dictionary: .github/rt-dictionary.yaml
+      - uses: newrelic/release-toolkit/next-version@v1
+        id: version
+        with:
+          tag-prefix: ${{  env.CHART_TAG_PREFIX }}
+          output-prefix: ""
       - name: Configure Git
         run: |
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
-      - name: Add newrelic repository
-        run: helm repo add newrelic https://helm-charts.newrelic.com
-
+      - uses: newrelic/release-toolkit/update-markdown@v1
+        with:
+          version: ${{ steps.version.outputs.next-version }}
+          markdown: ${{ env.CHART_DIRECTORY }}/CHANGELOG.md
+      - name: Commit updated changelog
+        run: |
+          yq e -i '.version="${{ steps.version.outputs.next-version }}"' charts/newrelic-prometheus-agent/Chart.yaml
```

Comment:
We can use the `CHART_DIRECTORY` env variable here.

Reply:
true!